### ISSUE TITLE
[Bug] r.mess runs on command line but not from Python console

### ISSUE THREAD

**Issue by ecodiv** [OSGeo/grass-addons#1252]
**Name of the addon**
r.mess

**Describe the bug**
The r.mess works when run from the command line. However, when run from the Python console, it throws an immediate error.

```
Traceback (most recent call last):
  File "/usr/lib/python3.12/code.py", line 90, in runcode
    exec(code, self.locals)
  File "<input>", line 1, in <module>
  File "/usr/lib/grass84/etc/python/grass/script/core.py", line 476, in run_command
    ps = start_command(*args, **kwargs)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/grass84/etc/python/grass/script/core.py", line 431, in start_command
    return Popen(args, **popts)
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/grass84/etc/python/grass/script/core.py", line 72, in __init__
    subprocess.Popen.__init__(self, args, **kwargs)
  File "/usr/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.12/subprocess.py", line 1868, in _execute_child
    for k, v in env.items():
                ^^^^^^^^^
AttributeError: 'str' object has no attribute 'items'
```

**To Reproduce**

In the Python console, run e.g.,

```
gs.run_command("r.mess", env="bio_1,bio_2", output="test_02")
```

**Expected behavior**
It should run, as it does on the command line, and generates the mess layers

**System description (please complete the following information):**

Operating System: 
Ubuntu 24.04
GRASS GIS version: 8.4.0
date=2024
revision=exported
build_date=2024-07-29
build_platform=x86_64-pc-linux-gnu
build_off_t_size=8
libgis_revision=8.4.0
libgis_date=2024-07-29T16:24:27+00:00
proj=9.4.1
gdal=3.9.1
geos=3.12.2
sqlite=3.45.1
Python=3.12.3

**Comment by ecodiv** [comment]
Seems to be solved with #1253

**Comment by ecodiv** [comment]
Solved in #1253
